studiopress/genesis-custom-blocks
# Update the color of the GCB Pro link for WP 5.7

#### Changes
* Changes the color of the 'Genesis Pro' submenu item to match the color in WP Core `5.7-beta1`
### Before with WP Core `5.6`
![before-link-gen-pro](https://user-images.githubusercontent.com/4063887/106831161-0a89fa00-6655-11eb-80d3-28721c00b0c3.png)

### After with WP Core `5.6`
![after-link-gen-pro](https://user-images.githubusercontent.com/4063887/106831130-fc3bde00-6654-11eb-8ea9-2ca73a0dc20e.png)

### After with WP Core `5.7-beta1`
![the-one-with-5 7](https://user-images.githubusercontent.com/4063887/106831627-d4994580-6655-11eb-80c3-b77a47e24669.png)

#### Testing instructions
* Go to `/wp-admin` 
* Sanity-check the blue color of `/wp-admin` > Genesis Blocks > Genesis Pro
